gcovr/gcov.py

```diff
@@ -17,7 +17,7 @@
 from .utils import aliases, search_file, Logger
 
 output_re = re.compile("[Cc]reating [`'](.*)'$")
-source_re = re.compile("[Cc]annot open (source|graph) file")
+source_re = re.compile("[Cc][annot|ould not] open (source|graph|output) file")
```

Comment:
I think this needs to use a regex group, not a character class for `[annot|ould not]` → `(annot|ould not)`.